test/cmdlineTests/yul_unimplemented/err

```diff
@@ -1,5 +1,5 @@
-Error (1834): Unimplemented feature error in <FILENAME REMOVED>
- --> yul_unimplemented/input.sol:8:9:
+Error (6359): Return argument type string memory is not implicitly convertible to expected type (type of first return variable) bytes memory.
+ --> yul_unimplemented/input.sol:5:16:
   |
-8 |         x.f();
-  |         ^^^^^
+5 |         return type(test).name;
+  |                ^^^^^^^^^^^^^^^
```

Comment:
Wasn't this meant to test the command-line output in case of an `UnimplementedError`? If so, then I think you need to find a different code that triggers one :)

Reply:
ok, the new one seems to do the trick.

Reply:
You still need to update the test output though.